src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -143,10 +276,19 @@ class FilterPopup extends React.Component {
 FilterPopup.propTypes = {
   location: PropTypes.object,
   filters: PropTypes.array,
+  createAPIQuery: PropTypes.func,
+  updateIsLoadingState: PropTypes.func,
+  selectedFilters: PropTypes.object,
```

Comment:
We are missing the type check for `searchKeywords` here.

Reply:
Updated. Thanks.